packages/mux-player/src/template.ts

```diff
@@ -139,7 +140,7 @@ export const content = (props: MuxTemplateProps) => html`
       <media-poster-image
         part="poster"
         exportparts="poster, img"
-        src="${props.poster === '' ? false : props.poster ?? false}"
+        src="${!!props.poster ? props.poster : false}"
```

Comment:
I think it's correct but these are sometimes tricky changes

Reply:
Yeah pretty sure this one is safe.